src/openrct2/audio/Audio.cpp

```diff
@@ -218,8 +218,8 @@ static AudioParams audio_get_params_from_location(SoundId soundId, const CoordsX
     {
         if (viewport->flags & VIEWPORT_FLAG_SOUND_ON)
         {
-            int16_t vy = pos2.y - viewport->viewPos.x;
-            int16_t vx = pos2.x - viewport->viewPos.y;
+            int16_t vy = pos2.y - viewport->viewPos.y;
+            int16_t vx = pos2.x - viewport->viewPos.x;
```

Comment:
I'd prefer if you also switched these two lines around. That's what induced the mistake in the first place.